src/test/tests/operators/tessellate.py

```diff
@@ -0,0 +1,225 @@
+#                          quadratic_linear_quad
+#                          quadratic_quad
+#                          quadratic_hex
+#                          triquadratic_hex
+#              plots     - pc, mesh
+#              operators - tessellate, clip
+#
+#  Programmer: Eric Brugger
+#  Date:       July 24, 2020
+#
+#  Modifications:
+#
+# ----------------------------------------------------------------------------
+
+# Quadratic_triangle
+OpenDatabase(data_path("vtk_test_data/quadratic_triangle.vtk"))
+
+AddPlot("Pseudocolor", "x_c")
+AddPlot("Mesh", "mesh")
+DrawPlots()
+
+v = GetView3D()
+v.viewNormal = (0.200511, 0.543812, 0.814901)
+v.focus = (0, 0.5, 1)
+v.viewUp = (-0.232184, 0.834474, -0.499744)
+v.viewAngle = 30
+v.parallelScale = 1.5
+v.nearPlane = -3
+v.farPlane = 3
+v.imagePan = (0, 0)
+v.imageZoom = 1
+v.perspective = 1
+SetView3D(v)
+
+Test("quadratric_triangle_01")
+
+AddOperator("Tessellate", 1)
+DrawPlots()
+
+Test("quadratric_triangle_02")
+
+tess = TessellateAttributes()
+tess.chordError = 0.01
+SetOperatorOptions(tess, 0, 1)
+
+Test("quadratric_triangle_03")
+
+CloseDatabase(data_path("vtk_test_data/quadratic_triangle.vtk"))
+DeleteAllPlots()
+
+# Biquadratic_quad
+OpenDatabase(data_path("vtk_test_data/biquadratic_quad.vtk"))
+
+AddPlot("Pseudocolor", "x_c")
+AddPlot("Mesh", "mesh")
+DrawPlots()
+
+v = GetView3D()
+v.viewNormal = (0.200511, 0.543812, 0.814901)
+v.focus = (0, 0.5, 1)
+v.viewUp = (-0.232184, 0.834474, -0.499744)
+v.viewAngle = 30
+v.parallelScale = 1.5
+v.nearPlane = -3
+v.farPlane = 3
+v.imagePan = (0, 0)
+v.imageZoom = 1
+v.perspective = 1
+SetView3D(v)
+
+Test("biquadratric_quad_01")
+
+AddOperator("Tessellate", 1)
+DrawPlots()
+
+Test("biquadratric_quad_02")
+
+tess = TessellateAttributes()
+tess.chordError = 0.01
+SetOperatorOptions(tess, 0, 1)
+
+Test("biquadratric_quad_03")
+
+CloseDatabase(data_path("vtk_test_data/biquadratic_quad.vtk"))
+DeleteAllPlots()
+
+# Quadratic_linear_quad
+OpenDatabase(data_path("vtk_test_data/quadratic_linear_quad.vtk"))
+
+AddPlot("Pseudocolor", "x_c")
+AddPlot("Mesh", "mesh")
+DrawPlots()
+
+v = GetView3D()
+v.viewNormal = (0.200511, 0.543812, 0.814901)
+v.focus = (0, 0.5, 1)
+v.viewUp = (-0.232184, 0.834474, -0.499744)
+v.viewAngle = 30
+v.parallelScale = 1.5
+v.nearPlane = -3
+v.farPlane = 3
+v.imagePan = (0, 0)
+v.imageZoom = 1
+v.perspective = 1
+SetView3D(v)
+
+Test("quadratric_linear_quad_01")
+
+CloseDatabase(data_path("vtk_test_data/quadratic_linear_quad.vtk"))
+DeleteAllPlots()
+
+# Quadratic_quad
+OpenDatabase(data_path("vtk_test_data/quadratic_quad.vtk"))
+
+AddPlot("Pseudocolor", "x_c")
+AddPlot("Mesh", "mesh")
+DrawPlots()
+
+v = GetView3D()
+v.viewNormal = (0.200511, 0.543812, 0.814901)
+v.focus = (0, 0.5, 1)
+v.viewUp = (-0.232184, 0.834474, -0.499744)
+v.viewAngle = 30
+v.parallelScale = 1.5
+v.nearPlane = -3
+v.farPlane = 3
+v.imagePan = (0, 0)
+v.imageZoom = 1
+v.perspective = 1
+SetView3D(v)
+
+Test("quadratric_quad_01")
```

Comment:
quadratric -> quadratic here and elsewhere

Reply:
Fixed spelling and renamed the test results. I also added the tests for the quadratic_mixed data.